package.json

```diff
@@ -26,7 +27,7 @@
     "@angular/flex-layout": "2.0.0-beta.7",
     "@angular/forms": "^4.0.0",
     "@angular/http": "^4.0.0",
-    "@angular/material": "2.0.0-beta.7",
+    "@angular/material": "^2.0.0-beta.7",
```

Comment:
flex-layout and material are mismatched. Check the flex-layout wiki and make sure we're using the correct version.